homeassistant/components/rotel/media_player.py

```diff
@@ -0,0 +1,493 @@
+"""
+Rotel platform.
+
+Although only the RSP-1570 is supported at the moment, the
```

Comment:
We prefer a 1-line header now -- perhaps, `Rotel RSP-1570 Media Player platform.`